app/config/config.js

```diff
@@ -0,0 +1,57 @@
+"use strict";
+
+const localEnv = _loadLocalEnv(),
+    env = process.env.NODE_ENV || 'development';
+
+
+/**
+ * @property {string} HOST
+ * @property {string} PORT
+ * @property {string} MONGODB_URL
+ */
+const config = {
+
+    development: {
+        HOST: localEnv.HOST || '188.166.63.248',
+        PORT: localEnv.PORT || '9030',
+
+        MONGODB_URL: localEnv.MONGODB_URL || 'mongodb://localhost:27017/differ-canvas'
+    },
+
+    test: {
+        HOST: '127.0.0.1',
+        PORT: '3000',
+
+        MONGODB_URL: 'mongodb://localhost:27017/differ-canvas_test_integration'
+    },
+
+    production: {
+        HOST: '127.0.0.1',
+        PORT: '9010',
+
+        MONGODB_URL: 'mongodb://localhost:27017/differ-canvas'
+    }
+};
+
+
+module.exports = config[env];
+
+
+/**
+ * @type function
```

Comment:
Yeah totally. We prefer reading the code over reading the jsdoc, but only if the functions are self-explanatory. 

Like instead of doing this:
```
router.get('/courses',
    CanvasMiddle.validateGetCoursesArgs, (req, res, next) => {

        CanvasSVC.getCourses(req, res, next);
    });
```

By separating the validation-logic entirely, we would probably prefer having the actual validation here, but by using a helper-method. Not saying your separation of concern is bad or whatever, it's just a different way of developing. Instead of separating code this much, we would rather use more helper-functions:

```
router.get('/courses',
    validateQuery(Joi.string().max(100).required()),  // if validating the query-string
     validateBody({ // if validating body
           something: joi.string()....
        }), 
        (req, res, next) => {
        CanvasSVC.getCourses(req, res, next);
    });
```

No need to refactor anything now though :)